packages/loot-core/src/server/importers/ynab5.ts

```diff
@@ -327,3 +327,19 @@ export function parseFile(buffer: Buffer): YNAB5.Budget {
 export function getBudgetName(_filepath: string, data: YNAB5.Budget) {
   return data.budget_name || data.name;
 }
+
+function equalsIgnoreCase(stringa: string, stringb: string): boolean {
+  return (
+    stringa.localeCompare(stringb, undefined, {
+      sensitivity: 'base',
+    }) === 0
+  );
+}
+
+function findByNameIgnoreCase(categories: YNAB5.CategoryGroup[], name: string) {
+  return categories.find(cat => equalsIgnoreCase(cat.name, name));
+}
+
+function findIdByName(categories: YNAB5.CategoryGroup[], name: string) {
+  return findByNameIgnoreCase(categories, name).id;
```

Comment:
We need to handle for null here:
```suggestion
  return findByNameIgnoreCase(categories, name)?.id;
```